ui/packages/statement/src/components/HorizontalBar.module.less

```diff
@@ -0,0 +1,47 @@
+.container {
+  display: flex;
+  align-items: center;
+}
+
+.text_container {
+  width: 64px;
+}
+
+.bar_container {
+  position: relative;
+}
+
+.normal_bar {
+  height: 14px;
+  background-color: #3a7de1;
+}
+
+.max_bar {
+  height: 3px;
+  background-color: #f2be2c;
```

Comment:
prefer to use ant design color palette: https://github.com/ant-design/ant-design/blob/master/components/style/color/colors.less

Reply:
Ok!

Reply:
color name gold-5 looks good to me